spec/index.html

```diff
@@ -1897,7 +2016,11 @@ <h3>Plain Literal Event</h3>
             class="arrow">·</span></a> accessor
             and "&quot;" (1 double quote).</p>
 
-          <p>Otherwise the value is the concatenation of "&quot;" (1 double quote),
+          <p>Otherwise, if the <a href="#eventterm-literal-literal-direction"
+            class="termref"><span
+            class="arrow">·</span>literal-direction<span
+            class="arrow">·</span></a> is the empty string,
+            the value is the concatenation of "&quot;" (1 double quote),
```

Comment:
`"&quot;" (1 double quote)` gets rendered as `""" (1 double quote)`. There are a number of possible ways to better present this (which should be applied to all similar instances). Probably the simplest would be ```<code>&quot;</code> (one double-quote)```.

Reply:
I'm not happy with this general way of emitting triples (see #55), and more could be done on character presentation, similar to how it's done in Turtle. This just adapts the existing pattern; I think another pass is needed separately to clean up these things. Just changing one of them because it happened to be touched by this PR doesn't really address the problem.

Reply:
Pick up in https://github.com/w3c/rdf-star-wg/issues/146